src/style.css

```diff
@@ -2,0 +1,105 @@
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc') format('truetype');
+  font-weight: 400;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.form {
+  font-family: Avenir, sans-serif;
+  display: block;
+  margin-top: 20px;
+  outline: none;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  width: 100%; /* Розтягує форму на всю ширину контейнера */
+  box-sizing: border-box; /* Включає відступи та межі в ширину */
+}
+
+.big {
+  font-size: 16px;
+  background: url('./images/loop.svg') no-repeat;
+  background-size: 19px;
+  box-sizing: border-box;
+  background-position: 26px center;
+  height: 70px;
+  padding-left: 62px;
+  background-color: #ffff;
+}
```

Comment:
The '.big' class seems to be used for a search input, but it's not clear from the class name. Consider renaming the class to something more descriptive, like '.search-input-large', to represent its purpose rather than its style.